app/controllers/api/v1/project_roles_controller.rb

```diff
@@ -10,9 +10,13 @@ def resource_name
   end
 
   def update
-    super do
-      UserAddedToProjectMailerWorker.perform_async(api_user.id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
-    end
+    UserAddedToProjectMailerWorker.perform_async(acl_user_id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
```

Comment:
Got you now on the ACL lookup [this](https://github.com/zooniverse/Panoptes/blob/7c0106054b20b4e13dc8e5f345784d8679b60e58/app/serializers/concerns/acl_serializer.rb#L31) is how we serialize the owner relation for that resource.